plugins/fastboot/fu-fastboot-device.c

```diff
@@ -623,6 +628,21 @@ fu_fastboot_device_write_firmware(FuDevice *device,
 				  GError **error)
 {
 	g_autoptr(FuFirmware) manifest = NULL;
+	g_autoptr(FuFirmware) flash_end_file = NULL;
+	FuFastbootDevice *self = FU_FASTBOOT_DEVICE(device);
+
+	/* fibocom modem Set the fibocom_flash_end value */
+	if (fu_device_has_vendor_id(self, "USB:0x2CB7")) {
+		flash_end_file = fu_firmware_get_image_by_id(firmware, "flash_end", NULL);
+		manifest = fu_firmware_get_image_by_id(firmware, "partition_nand.xml", NULL);
+		if (flash_end_file != NULL && manifest == NULL) {
+			self->fibocom_flash_end-=1;
```

Comment:
formatting is funky -- have you run ./contrib/setup and set up precommit? clang-format should have fixed this..